### ISSUE TITLE
Update matrix

### ISSUE THREAD

**Issue by frewsxcv** [kenn/app_store_pricing_matrix#5]
A new matrix just came out, would be nice to have the matrix in here updated

**Comment by jstart** [comment]
For anyone who ends up here through a failed Google (like I did) there is now a private JSON API to get the pricing matrix:

https://itunesconnect.apple.com/WebObjects/iTunesConnect.woa/ra/apps/458272450/pricing/matrix

Gist with the JSON as of today.
https://gist.github.com/jstart/822c122d12420d46703f

**Comment by kenn** [comment]
Does not work for me. How do you get the credentials?

```json
{"data":null,"messages":{"warn":null,"error":["Unauthorized access"],"info":null},"statusCode":"ERROR"}
```

**Comment by jstart** [comment]
@kenn you need to already have an authentication cookie for iTunes Connect, so log in first, then request the URL.

**Comment by kenn** [comment]
Nope.

```json
{"data":null,"messages":{"warn":null,"info":null,"error":["operation_failed"]},"statusCode":"ERROR"}
```

**Comment by jstart** [comment]
As mentioned previously, you have to be logged in to iTunes Connect to request the iTunes Pricing Matrix.  Is anyone on your team a member of iTunes Connect?  Do you need to set up a generic login that can hit the endpoint?

Christopher Lee Truman
iOS Developer - eHarmony
ctruman.info
Cell - (310) 347-9814
cleetruman@gmail.com

LinkedIn - http://www.linkedin.com/in/christophertruman
Github - http://github.com/jstart
Skype - Jstart
Twitter - @iAmChrisTruman

**Comment by kenn** [comment]
I'm an owner (agent, in Apple's term) of multiple iTunes Connect accounts, but the URL you gave here never worked for me when I'm logged in on itunesconnect.apple.com with any of those accounts. Can you be more specific?

**Comment by sowenjub** [comment]
@kenn You need to use the Apple ID of an app you own in the url

https://itunesconnect.apple.com/WebObjects/iTunesConnect.woa/ra/apps/{AppleID}/pricing/matrix

**Comment by kenn** [comment]
Thanks everyone! I've finally figured it out and released v3.0.0 which is a complete rework. Let me know what you think!

**Comment by kenn** [comment]
Fixed file loading problem with 3.0.1
